cypress/integration/channel/autocomplete_shown_each_channel_spec.js

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [number] indicates a test step (e.g. 1. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+describe('Identical Message Drafts', () => {
+    before(() => {
+        // 1. Login and go to /
+        cy.login('user-1');
+        cy.visit('/');
+    });
+
+    it('M14432 shows Autocomplete in each channel', () => {
+        // 1. Go to test Channel A on sidebar
+        cy.get('#sidebarItem_town-square').scrollIntoView();
+        cy.get('#sidebarItem_town-square').should('be.visible').click();
+
+        // * Validate if the channel has been opened
+        cy.url().should('include', '/ad-1/channels/town-square');
```

Comment:
Can we change this to `/channels/town-square` and remove team name from Url? This way we can minimise the overhead if we default team changes later.

Reply:
Can we do this for all the Url instances in the in tests here?

Reply:
That's a good point and I think it makes sense since the test spec is not specific for URL validation.  Agree on changing URL validation here.  For other URL instances not related to this prefer, I recommend a separate PR for it.